includes/admin/class-wcmypa-admin.php

```diff
@@ -917,6 +931,116 @@ private function printDeliveryDate(DeliveryOptions $deliveryOptions): void
     }
 
     /**
+     * Output the chosen delivery options or the chosen pickup options.
+     *
+     * @param DeliveryOptions $deliveryOptions
+     *
+     * @return array[]|null
+     * @throws \Exception
+     */
+    private function getConfirmationData(DeliveryOptions $deliveryOptions): ?array
+    {
+        $signatureTitle     = (new WCMP_Checkout)->getDeliveryOptionsTitle(WCMYPA_Settings::SETTING_SIGNATURE_TITLE);
+        $onlyRecipientTitle = (new WCMP_Checkout)->getDeliveryOptionsTitle(WCMYPA_Settings::SETTING_ONLY_RECIPIENT_TITLE);
+
+        if (! $deliveryOptions->getCarrier()) {
+            return null;
+        }
+
+        if ($deliveryOptions->getDeliveryType() === AbstractConsignment::DELIVERY_TYPE_PICKUP_NAME) {
+
+            return [
+                __("Delivery type:", "woocommerce-myparcel")   => WCMP_Data::getDeliveryTypesHuman()[$deliveryOptions->getDeliveryType()],
+                __("Pickup location:", "woocommerce-myparcel") =>
+                  $deliveryOptions->getPickupLocation()->getLocationName() . '<br>' .
```

Comment:
Kan je dit converteren naar sprintf? Phpstorm heeft daar een actie voor. Ik denk dat het het iets nicer maakt.

Reply:
misschien ook ff de pickup location in een variabele stoppen, scheelt een hoop leeswerk